src/app/A1-page-error/page-error.component.html

```diff
@@ -1 +1,11 @@
-<p>page-error works!</p>
+<h1 class="mt-32 text-4xl font-bold text-center">OOPS!</h1>
+
+<p class="text-2xl text-center font-light mt-12 px-8    ">
+Looks like something went wrong! The page you're looking for misplaced itself.
+</p>
+
+<div class="px-8 mt-10">
+    <button class="rounded-full text-white bg-gray-500 w-full py-4">
+        Report the Issue
```

Comment:
@theJayBecker @rorepopo We should talk with Robyne, about what we should, if we actually want to report an error, back to the system. See if her mobin has a treatment, we might want the UX to do, or happen.